server/plugin/command.go

```diff
@@ -347,18 +371,39 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 			}
 			subOrgMsg += "\n\n" + fmt.Sprintf("Notifications are disabled for %s", excludeMsg)
 		}
+
+		subscriptionSuccess := fmt.Sprintf("Github subscription \"[%s](%s)\" was added to this channel by @%v, with events: %s", owner, orgLink, user.Username, subscribeEvents.String())
+
+		if previouslySubscribed {
+			subscriptionSuccess += previouslySubscribedEventMessage
+			subOrgMsg += previouslySubscribedEventMessage
+		}
+
+		post := &model.Post{
+			ChannelId: args.ChannelId,
+			UserId:    p.BotUserID,
+			Message:   subscriptionSuccess,
+		}
+
+		if _, err := p.API.CreatePost(post); err != nil {
+			p.API.LogWarn("error while creating post", "post", post, "error", err.Error())
+			return fmt.Sprintf("%s Though there was an error creating the public post: %s", subscriptionSuccess, err.Error())
+		}
 		return subOrgMsg
 	}
 	if flags.ExcludeOrgRepos {
 		return "--exclude feature currently support on organization level."
 	}
 
-	if err := p.Subscribe(ctx, githubClient, args.UserId, owner, repo, args.ChannelId, features, flags); err != nil {
+	if err := p.Subscribe(ctx, githubClient, args.UserId, owner, repo, args.ChannelId, string(subscribeEvents), flags); err != nil {
 		return err.Error()
 	}
 	repoLink := p.getBaseURL() + owner + "/" + repo
 
-	msg := fmt.Sprintf("Successfully subscribed to [%s](%s).", repo, repoLink)
+	msg := fmt.Sprintf("Github subscription \"[%s](%s)\", was added to this channel by @%v, with events: %s", repo, repoLink, user.Username, subscribeEvents.String())
```

Comment:
Both confirmation messages should make it clear that the sub is either for a repository or an organization.